.github/workflows/deploy-production.yml

```
@@ -36,5 +36,6 @@ jobs:
           VITE_STRIPE_TOKEN_PROD: ${{ secrets.PROD_STRIPE_TOKEN }}
           VITE_RECAPTCHA_SITE_KEY: ${{ secrets.PROD_RECAPTCHA_SITE_KEY }}
           VITE_SEGMENT_TOKEN: ${{ secrets.PROD_SEGMENT_TOKEN }}
+          CROSS_EDGE_SECRET: ${{ secrets.CROSS_EDGE_SECRET}}
```

Todos os outros secrets estão com o prefixo `PROD_`. Por favor siga o padrão
```suggestion
          CROSS_EDGE_SECRET: ${{ secrets.PROD_CROSS_EDGE_SECRET}}
```